### PR TITLE
Used anugular2 forms for quick add work item

### PR DESCRIPTION
**Pull request by harrybabu**
Fixes for #139

### CODE REVIEW

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.html [hunk 1]
```diff
@@ -5,31 +5,37 @@
...
+                      (keydown.enter)="save($event)"
+                      (keyup.enter)="preventDef($event)"
+                      (keyup)="checkDesc()"
+                      placeholder="Description">
+              </textarea>
+            </div>
+        </div>
+        <div class="workItemQuickAdd_AddBlk">
+          <button type="submit" [disabled]="!validTitle" [ngClass]="{'icon-btn': validTitle,'icon-btn-disabled':!validTitle}" class="pull-left font16 workItemQuickAdd_Add">
+            <i class="fa fa-check font14"></i>
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.html]: The disabled state needs to be updated. The check icon can be grey in colour. Currently the blue check disappears.

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.html [hunk 2]
```diff
@@ -5,31 +5,37 @@
...
+          </button>
+          <button (click)="toggleQuickAdd()" class="pull-right closeQuickAdd"><i class="pficon pficon-close font16"></i></button>
```
Comment (nimishamukherjee) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.html]: When I click on code, the url looks like http://0.0.0.0:8088/work-item-list?title=Test%2BTitle&description=Desc
the title and description are appended to the URL

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.html (hunk 1 above)
Comment (harrybabu) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.html]: Working fine. Confirmed with @nimishamukherjee also.

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.html (hunk 2 above)
Comment (harrybabu) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.html]: Working fine. Confirmed with @nimishamukherjee.

**Comment by nimishamukherjee** [comment]
Reviewed the PR and LGTM.

**Comment by joshuawilson** [comment]
[test]

File: src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss [hunk 3]
```diff
@@ -55,7 +55,7 @@
       i{
         position: absolute;
         font-size:$font18;
-        top:50%;
+        top:100%;
```
Comment (joshuawilson) [inline src/app/work-item/work-item-quick-add/work-item-quick-add.component.scss]: Why did you change these buttons? The Mobile view does not look correct now.

**Comment by joshuawilson** [comment]
@nimishamukherjee Why did you want the quick add to be a form? It does not appear to add anything to it by making it a form.

**Comment by nimishamukherjee** [comment]
@joshuawilson primarily for consistency. We can drop this if needed

**Comment by joshuawilson** [comment]
These changes break the UI and adding Forms to this component does not provide value. Therefore, I'm closing it.
